dataservice/api/participant/schemas.py

```diff
@@ -6,20 +6,35 @@
 
 
 class ParticipantSchema(BaseSchema):
-    # Should not have to do this, since participant_id is part of the
-    # Demographic model and should be dumped. However it looks like this is
-    # still a bug in marshmallow_sqlalchemy. The bug is that ma sets
-    # dump_only=True for foreign keys by default. See link below
-    # https://github.com/marshmallow-code/marshmallow-sqlalchemy/issues/20
     study_id = field_for(Participant, 'study_id', required=True,
                          load_only=True)
+    family_id = field_for(Participant, 'family_id',
+                          required=False,
+                          load_only=True, example='FM_ABB2C104')
 
     class Meta(BaseSchema.Meta):
         model = Participant
         resource_url = 'api.participants'
         collection_url = 'api.participants_list'
+        exclude = ('study', 'family')
 
     _links = ma.Hyperlinks({
         'self': ma.URLFor(Meta.resource_url, kf_id='<kf_id>'),
-        'collection': ma.URLFor(Meta.collection_url)
+        'collection': ma.URLFor(Meta.collection_url),
+        'study': ma.URLFor('api.studies', kf_id='<study_id>'),
     })
+
+    def dump(self, p, *args, **kwargs):
```

Comment:
Sigh its unfortunate that this is still an issue in flask-ma [.](https://github.com/marshmallow-code/flask-marshmallow/issues/68) I tried looking at the source code to see if we can handle this in a more generic way and thought of a potential fix. However, I don't think its worth it to implement bc i'm unsure of how the fix could affect query params when we decide to implement that in our API.